src/main/flight/interpolated_setpoint.c

```diff
@@ -102,11 +102,11 @@ FAST_CODE_NOINLINE float applyFfLimit(int axis, float value, float Kp, float cur
         DEBUG_SET(DEBUG_FF_LIMIT, 0, value);
     }
 
-    if (axis == FD_ROLL) {
+    if (axis == FD_PITCH) {
         DEBUG_SET(DEBUG_FF_LIMIT, 1, value);
     }
 
-    if (ffMaxRateLimit[axis]) {
+    if (ffMaxRateLimit[axis] && (axis < FD_YAW)) {
```

Comment:
This can be done in a more efficient way: `applyFfLimit()` is used in the following way: 

```
pidData[axis].F = shouldApplyFfLimits(axis) ?
                applyFfLimit(axis, feedForward, pidCoefficient[axis].Kp, currentPidSetpoint) : feedForward;
```

So changing `shouldApplyFfLimits()` to return 

```
ffMaxRateLimit[axis] != 0.0f && axis < FD_YAW
```

will get the same effect, without having to call into `applyFfLimit()` for `FD_YAW`.

Also checking floats for 0 should be done as `ffMaxRateLimit[axis] != 0.0f`.